pkg/util/sys/parse.go

```diff
@@ -22,13 +22,13 @@ import (
 
 // grep finds the *first* line that matches, rather than multiple lines
 func Grep(input, searchFor string) string {
-	logger.Debugf("grep. search=%s, input=%s", searchFor, input)
+	logger.Debugf("grep. searching")
```

Comment:
What about keeping the `search=%s` but remove the input? So we still know the pattern we are looking for.

Reply:
mmm... is a very general purpose function. What if in the input we put sensitive information ....it seems a little bit dangerous. I think that better not run with the risk.

Reply:
What's typically in the input?

Reply:
As is stated in the bug #4568 :
`The ​rook/pkg/operator/ceph/cluster/mon.extractKey()​ function attempts to parse secrets from arbitrary string inputs by using the r​ook/pkg/util/sys.Grep()`

The function extractKey uses the keyring file to parse keys.... does not seem good idea to log the whole content of the keyring file.

But in any case, Grep is a general use function , and in the future it can be used in other ways and the log of the "input" string also can drive us to generate new security issues, so i think that better not to log  information about the search , only if we execute the search and if we have found something. ( as debug clue).

Reply:
Well if you don't debug anything useful than simply printing a debug then we should probably remove these debug lines altogether...

Reply:
removed